### PR TITLE
Feature UNO 467 - Extend Response Properties on text Entry interaction with possibility to set baseType property 

### PR DESCRIPTION
**Pull request by habemuscode**
**Needs to be merged and publish before this PR**

https://github.com/oat-sa/tao-item-runner-qti-fe/pull/96

**Related to**

https://oat-sa.atlassian.net/browse/UNO-467

**Description**

**As** content creator
**I would** like during the assessment on textEntry interaction and in case of touch-based devices, to open the numeric keyboard for input
**so that**  we increase user experience and eliminate possibilities for wrong input

**Acceptance Criteria**

**AC1**

**GIVEN** an inline `textEntry` interaction flagged as expecting a numeric input
**WHEN** the test taker tap on the input field with a touch-based device like a tablet
**THEN** the device opens the numeric keyboard instead of the alphanumeric keyboard

**Considerations**

The main goal is to implement this story on the NextGen test runner.

- NextGen test runner

Implementation of NextGen test runner already support such behavior.  We have to add inputmode=numeric if the `baseType` is integer and `inputmode=decimal` if the `baseType` is a `float`

- CurrentGen test runner

Implementation for CurrentGen test runner implement similar logic as on NextGen test runner relying on `inputmode`

**How to Test**

**Requires touch-based device, tablet or similar**

Create an item with `textEntry` interaction with Response `baseType` declaration set to integer or float.

Create a test including the item(s) created before and publish to NextGen to deliver an environment that is linked with New Test runner ex.

on demo.udir publish on using tenant: UDIR QA (new Test Runner)

Launch delivery execution from the touch-based device

Verify during input on textEntry interaction with numeric `baseType`,  numeric keyboard is rendered.

### CODE REVIEW

**Comment by adriloma** [comment]
Seems that some Jenkins job is failing.

**Review by andrey-ponamarev — CHANGES_REQUESTED** [review]
Please update package lock file

**Comment by habemuscode** [comment]
@andrey-ponamarev  @adriloma This PR needs the other merge before (check description). I just created to be more fast merging, instead of add people after first was merge. I can get approvals before as long as is just version update

**Review by bziondik — APPROVED** [review]
Add lock file
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Comment by habemuscode** [comment]
@andrey-ponamarev Lock file pushed

**Review by jsconan — APPROVED** [review]
Code review only.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**
